tests/dramatiq/conftest.py

```diff
@@ -0,0 +1,37 @@
+import dramatiq
+import pytest
+from dramatiq.brokers.stub import StubBroker
+from dramatiq.results import Results
+from dramatiq.results.backends import StubBackend
+from pytest_mock_resources import create_redis_fixture
+
+from strapp.dramatiq import configure
+
+redis = create_redis_fixture()
+
+
+@pytest.fixture()
```

Comment:
should these be in testing.py? as functions to be called by an end-user to produce their own fixture, a la pmr?